src/collective/taxonomy/configure.zcml

```diff
@@ -9,8 +9,6 @@
     xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="collective.taxonomy">
 
-  <includeDependencies package="." />
-
   <i18n:registerTranslations directory="locales" />
 
   <include package="plone.app.registry" />
```

Comment:
I'd add 
```
<include package="plone.supermodel" />
```
here and remove `plone.app.dexterity` in setup.py, because there's no dependency on that ...